invoke/monkey.py

```diff
@@ -1,6 +1,9 @@
 # Fuckin' A.
 
-import select, errno, os, sys
+import errno
```

Comment:
Unclear why this file is being touched, it's gone in HEAD. Can you try rebasing on latest HEAD perhaps?